cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js

```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+
+context('Click SaveToCloud buttons', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('should toggle GoogleDrive SaveToCloudButton', () => {
+    cy.get('#root > div > aside > div > div:nth-child(3) > div:nth-child(2) > button.ui.yellow.big.icon.button').click();
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.content > div > div > p').contains('Google Drive')
+    cy.get('body > div.ui.top.aligned.page.modals.dimmer.transition.visible.active > div > div.actions > button').click();
```

Comment:
You probably want to remove this last click from the test.

Reply:
Updated